examples/src/TabbableVoidsApp.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+  createListPlugin,
+  createTabbableVoidsPlugin,
+  ELEMENT_CODE_BLOCK,
+  ELEMENT_LI,
+  findNode,
+  Plate,
+} from '@udecode/plate';
+import { basicNodesPlugins } from './basic-nodes/basicNodesPlugins';
+import { editableProps } from './common/editableProps';
+import { plateUI } from './common/plateUI';
+import { createTabbableVoidsElementPlugin } from './tabbable-voids/createTabbableVoidsElementPlugin';
+import { TabbableVoidsElement } from './tabbable-voids/TabbableVoidsElement';
+import { tabbableVoidsValue } from './tabbable-voids/tabbableVoidsValue';
+import { createMyPlugins, MyValue } from './typescript/plateTypes';
+
+const plugins = createMyPlugins(
+  [
+    ...basicNodesPlugins,
+    createListPlugin(),
+    createTabbableVoidsPlugin({
+      options: {
+        query: (editor) => {
+          const inList = findNode(editor, { match: { type: ELEMENT_LI } });
+          const inCodeBlock = findNode(editor, {
+            match: { type: ELEMENT_CODE_BLOCK },
+          });
+          return !inList && !inCodeBlock;
+        },
+      },
+    }),
+    createTabbableVoidsElementPlugin({
+      component: TabbableVoidsElement,
+    }),
+  ],
+  {
```

Comment:
Could we add the indent plugin here to highlight its support?

Reply:
Good idea! I forgot about the indent plugin

Reply:
Looking at the indent plugin, I don't think there's any good way of supporting it alongside tabbable-voids, unless you can think of a way to determine the user's intent when they press tab.

If developers do need to support the indent plugin alongside this one, perhaps they could use the `query` option to only enable tabbable-voids when `editor.selection` is equal to the path of the void you're tabbing into. I can make sure that works and add it to the docs when I write them.